source/adminguide/networking.rst

```diff
@@ -318,13 +320,13 @@ To create a network offering:
                           virtual router or any other Port Forwarding providers that have
                           been configured in the cloud.
       VPN                 For more information, see `“Remote Access                                    Supported     Not Supported
-                          VPN” <networking2.html#remote-access-vpn>`_.
+                          VPN” <networking_and_traffic.html#remote-access-vpn>`_.
       User Data           For more information, see `“User Data and Meta                               Not Supported Supported
                           Data” <api.html#user-data-and-meta-data>`_.
-      Network ACL         For more information, see `“Configuring Network Access                       Supported     Not Supported
-                          Control List” <networking2.html#configuring-network-access-control-list>`_.
+      Network ACL         For more information, see `“Configuring Network Access Control               Supported     Not Supported
+                          List” <networking_and_traffic.html#configuring-network-access-control-list>`_.
       Security Groups     For more information, see `“Adding a Security                                Not Supported Supported
-                          Group” <networking2.html#adding-a-security-group>`_.
+                          Group” <networking_and_traffic.html#adding-a-security-group>`_.
       =================== ============================================================================ ============= =============
```

Comment:
this table has dissappeared ???
![image](https://user-images.githubusercontent.com/2486961/106865270-d77e4f80-66c2-11eb-9794-7a7023a62232.png)

Reply:
@DaanHoogland should be fixed now.

PS: I had to clear cache for the page to see changes